README.md

```diff
@@ -6,14 +6,14 @@ This repo contains the installer prototype for FlyByWire Simulations projects.
 
 ## Build dependencies
 
-Please make sure you have: 
+Please make sure you have:
 - NodeJS 14 - [Homepage](https://nodejs.org/en/)
 
 ## How to open
 
 First install the dependencies
 ```shell script
-npm install
+npm ci
```

Comment:
Undo this change

Reply:
It's better for compatibility, to install the already preconfigured dependencies initially via ``npm ci`` (that's why you commit package-lock.json) and only use ``npm install`` when you intend to update or add dependencies, but if you want me to, sure.